examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/airflow_dags/dags.py

```diff
@@ -8,7 +8,6 @@
 from airflow.operators.bash import BashOperator
 from dagster_airlift.in_airflow import mark_as_dagster_migrating
 from dagster_airlift.migration_state import load_migration_state_from_yaml
-
```

Comment:
there appears to be some sort of war going on here: https://github.com/dagster-io/dagster/pull/23937/files#diff-9bfcb99ddb085c42daebaa9c85338c24cfbf66a13bc1c7f23b54a2a319727027R11

Reply:
my local `make ruff` appears to prefer the extra newline fwiw

Reply:
very odd -- i reinstalled ruff and same thing, so i'll just let the war continue to wage